app/invocation/invocation_cache_card.tsx

```diff
@@ -278,7 +278,7 @@ function renderBreakdown(data: any[] | undefined, title: string, subtitle: strin
 
   let sum = data?.reduce((prev, current) => {
     return { name: "Sum", value: prev.value + current.value };
-  }, 0);
+  }, { name: "Sum", value: 0 });
 
   let cap = 5;
   let other = 0;
```

Comment:
will `format.percent` throw an error if `sum.value` turns out to be 0? (let's test this locally before submitting)

Reply:
nvm I see, we won't access `sum.value`